.github/workflows/release-packages.yaml

```diff
@@ -225,5 +225,5 @@ jobs:
         run: docker build -t "$IMAGE_TAG" .
       - name: Push docker image to DockerHub
         run: |
-          docker login --username=dbcicprover --password=${{ secrets.DOCKERHUB_ACCESS_DB_CI_CPROVER }}
+          echo ${{ secrets.DOCKERHUB_ACCESS_DB_CI_CPROVER }} | docker login --username=dbcicprover --password-stdin
```

Comment:
I presume this works same way as `docker login --username=dbcicprover --password-stdin < echo ${{ secrets.DOCKERHUB_ACCESS_DB_CI_CPROVER }}` so LGTM